pyspecProcScripts/phasing.py

```diff
@@ -423,38 +423,3 @@ def hermitian_function_test(
     return best_shift
 
 
-def determine_sign(s, direct="t2", fl=None):
```

Comment:
What are you doing here? This doesn't have to do with the log, so why mess with this?